Joe's HW7 file

puleo-joe opened the pull request:
@MengyuanZoe @hanghu @YangJiang89 My HW7 file for editing.

JoePuleo committed:
Joe's HW7 file

hanghu commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+from folium import plugins
+
+#load King County datasets
+crimes = pd.read_csv('King_County_Sheriff_s_Office.csv')
```
the data is not in the same folder, you might want to add the path of the data files.

hanghu commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+
+#import folium plugins
+from folium.plugins import MarkerCluster
```
unused modules MarkerCluster and os

hanghu commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+for name, row in houses[0:200].iterrows():
+    folium.Marker([row["lat"], row["long"]],
+                  popup="Price: {0}$, Size: {1} sqft, Bedrooms: {2}, Bathrooms: {3}, Year Built: {4}"\
+                  .format(row["price"], row['sqft_living'], row['bedrooms'], row['bathrooms'], row['yr_built'])).add_to(house_groups)
```
the above two lines are too long

hanghu commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+    plugins.HeatMap([[row["latitude"], row["longitude"],row["city"]]
+    for iters, row in crimes[0:100].iterrows()],
+    min_opacity=1, max_zoom=10, radius = 8))
+
```
The indentation of above two lines are wrong

hanghu commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+#save output of house markers
+kingcounty_map.save('houses_map.html')
+
+'''Create new basemap for crimes heat map'''
```
for one line comments, a simple "#" before the comments could work too

YangJiang89 commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+kingcounty_map.save('houses_map.html')
+
+'''Create new basemap for crimes heat map'''
+kccrimes_heatmap = folium.Map(location = [houses['lat'].mean(), houses['long'].mean()], zoom_start = 10)
```
no white space around '=' as the function parameter argument

YangJiang89 replied:
Line too long

YangJiang89 commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+#prepare base map
+'''Use mean of latitude and longitude in dataset and set original zoom'''
+kingcounty_map = folium.Map(location = [houses['lat'].mean(),
+	houses['long'].mean()], zoom_start = 10)
```
Wrong continued indentation

YangJiang89 commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+
+'''Add data to heat map and select opacity'''
+kccrimes_heatmap.add_children(
+    plugins.HeatMap([[row["latitude"], row["longitude"],row["city"]]
```
white space after comma

YangJiang89 commented on HW7/JPHW7.py:
```diff
@@ -0,0 +1,55 @@
+kccrimes_heatmap.add_children(
+    plugins.HeatMap([[row["latitude"], row["longitude"],row["city"]]
+    for iters, row in crimes[0:100].iterrows()],
+    min_opacity=1, max_zoom=10, radius = 8))
```
no white space around '=' as the function parameter argument

JoePuleo committed:
Updated HW7

Removed whitespace, shortened lines, deleted unused imports, corrected file paths, and corrected indenting.
